### ISSUE TITLE
:visited pseudo class no longer works [regression]

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liudonghua123/ie7-js#232]
```
The :visited pseudo class was wrongly removed.
```

Original issue reported on code.google.com by `dean.edw...@gmail.com` on 15 Mar 2010 at 12:05

**Comment by GoogleCodeExporter** [comment]
Original comment by `dean.edw...@gmail.com` on 25 Apr 2010 at 2:54
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 2.1 beta4.
```

Original comment by `dean.edw...@gmail.com` on 30 Apr 2010 at 9:14
- Changed state: **Fixed**
